Userland/Libraries/LibJS/Runtime/Temporal/ISO8601.cpp

```diff
@@ -900,10 +926,25 @@ bool ISO8601Parser::parse_time_zone_iana_name_tail()
 bool ISO8601Parser::parse_time_zone_iana_name()
 {
     // TimeZoneIANAName :
-    //     TimeZoneIANANameTail
+    //     Etc/GMT ASCIISign UnpaddedHour
+    //     TimeZoneIANANameTail but not Etc/GMT ASCIISign UnpaddedHour
```

Comment:
That's a bit weird :thinking: what's the point of excluding a subset of a production and then immediately allowing it again

Reply:
Yeah, no idea :man_shrugging:

Reply:
Perhaps the author thought sub-productions need to mutually exclusive, even though ones that are not already exist...